.vsts-dotnet-ci.yml

```diff
@@ -58,6 +58,11 @@ jobs:
     inputs:
       filename: 'eng/cibuild_bootstrapped_msbuild.cmd'
       arguments: '-msbuildEngine dotnet'
+  - task: RichCodeNavIndexer@0
```

Comment:
Was it intentional to only add this to the Core on Windows build?

Reply:
Honestly, mostly just picked a build :) We have a requirement to run on Windows, but otherwise no reason to do "Windows Core" over "Windows Full"